docs/internals/contributing/bugs-and-features.txt

```diff
@@ -110,10 +110,10 @@ part of that. Here are some tips on how to make a request most effectively:
 If there's a consensus agreement on the feature, then it's appropriate to
 create a ticket. Include a link to the discussion in the ticket description.
 
-As with most open-source projects, code talks. If you are willing to write the
-code for the feature yourself or, even better, if you've already written it,
-it's much more likely to be accepted. Fork Django on GitHub, create a feature
-branch, and show us your work!
+As with most open-source projects, code talks. However, before writing any code,
+it's important to follow our updated process. If you have a bug fix or feature request,
```

Comment:
"Updated process" seems a bit distracting to me, since it draws attention to something changing without saying what changed. I wonder if we can replace this paragraph with a single sentence at the end of the previous one reminding folks that it is possible to either 1.) include diffs in tickets, or 2.) link to a draft PR -- the point to reinforce being that ideally the discussion starts on the ticket before moving to a PR.

Reply:
Thank you for reviewing. 

I am unsure what updates are needed here, and I find the current paragraph a bit confusing. As I understand it, it says you can directly create a feature request,  should we clarify that you need to create a ticket on Trac before proceeding with that? Something like this:

> As with most open-source projects, code talks. If you are willing to write the
> code for the feature yourself or, even better, if you've already written it,
> it's much more likely to be accepted. Ensure you create a ticket on track first , and show us your work!

Reply:
We have a few options. On Trac someone suggested just deleting the paragraph. That makes sense to me, since it does just largely repeat the generic encouragement at the top of this document, only for features instead of bugs: "bug reports with associated contributions will get fixed far more quickly than those without a solution.")

_If_ this paragraph has enough value to remain in some fashion, I think at best it reminds folks that showing a draft implementation can help others evaluate an idea. (My last comment suggested possibly reminding people to use draft PRs or embedding a diff in a ticket.) But just above this there is already "Include example code (non-functional is OK) if possible." I think people comfortable with GitHub will understand there are multiple ways to do this, with or without an actual PR, without us needing to detail it any further.

Others may have useful input here, but I'm leaning toward just deleting this paragraph. What do you think?
***
> should we clarify that you need to create a ticket on Trac

I think we might muddle things further if we just say that. The gist of the portion above this is all about encouraging people to go to the forum with any hefty feature request and waiting to open a ticket only if consensus develops.

Reply:
Yes, I also agree with deleting the paragraph, as it repeats what was said before. I will go ahead and delete it.

Reply:
I have updated the change requested. You can review it again. Thank you.